manifests/config.pp

```diff
@@ -66,6 +66,7 @@
   $proxy_buffers                  = '32 4k',
   $proxy_buffer_size              = '8k',
   $proxy_cache_inactive           = '20m',
+  $proxy_cache_key                = '$scheme$proxy_host$request_uri',
```

Comment:
$scheme$proxy_host$request_uri is the actual default value:
http://nginx.org/en/docs/http/ngx_http_proxy_module.html#proxy_cache_key